src/sonic-config-engine/portconfig.py

```diff
@@ -376,7 +378,8 @@ def get_config(self):
                     'alias': self._breakout_capabilities[alias_id],
                     'lanes': ','.join(lanes),
                     'speed': str(entry.default_speed),
-                    'index': self._indexes[lane_id]
+                    'index': self._indexes[lane_id],
+                    'subport': "0" if total_num_ports == 1 else str(alias_id + 1)
```

Comment:
I got from HLD https://github.com/sonic-net/SONiC/blob/master/doc/dynamic-port-breakout/sonic-dynamic-port-breakout-HLD.md that subport number should be 0 if there is no breakout, but it will start from 1 if there is breakout. What is the benefit of it? Why not always set subport to 1 for the first port no matter there is breakout or not?
```
Configure a unique subport# for each broken-down (logical) port in platform's port_config.ini

subport# to start with 1 (and increment sequentailly for each logical port) under the same physcial port
subport# sequence may repeat for logical ports under another physcial port
subport# as 0 (on a port) implies physical port itself (i.e. no port breakout on it)
```

Reply:
@Junchao-Mellanox we want to keep it simple for Xcvrd to distinguish between a breakout port (subport >= 1) vs a non-breakout port(subport = 0)